src/plugins/home/public/application/components/home.tsx

```diff
@@ -91,15 +111,16 @@ export class Home extends Component {
 
   skipWelcome = () => {
     this.props.localStorage.setItem(KEY_ENABLE_WELCOME, 'false');
-    this._isMounted && this.setState({ isWelcomeEnabled: false });
+    if (this._isMounted) this.setState({ isWelcomeEnabled: false });
   };
 
-  findDirectoryById = (id) => this.props.directories.find((directory) => directory.id === id);
+  findDirectoryById = (id: string) =>
```

Comment:
optional nit: `findDirectoryById` and `getFeaturesByCategory` aren't passed by reference, so no need to bind them to every component instance.
```suggestion
  findDirectoryById (id: string) {
```